deploy/template.yaml

```diff
@@ -120,6 +120,22 @@ Mappings:
       RESOURCETTLSECS: 1209600 # Default 14 days
       CLIENTS: '[{"jwksEndpoint":"https://api.identity.staging.account.gov.uk/.well-known/jwks.json","clientId":"03A5A659-17AA-453F-B905-95D2804823D1","redirectUri":"https://identity.staging.account.gov.uk/credential-issuer/callback?id=bav"}]'
       AUTHSESSIONTTLSECS: 950400 # 11 days in seconds
+    integration:
+      ISSUER: 'https://review-bav.integration.account.gov.uk'
+      DNSSUFFIX: review-bav.integration.account.gov.uk
+      MESSAGERETENTIONPERIODDAYS: 345600 # Default: 4 days
+      TXMAMESSAGERETENTIONPERIODDAYS: 604800 # Default: 7 days
+      RESOURCETTLSECS: 1209600 # Default 14 days
```

Comment:
Will the resource TTL value be reviewed and changed as needed as a separate task/story. This value is 14 days and will not be valid for BAV as the session will be short lived.

Reply:
This environment variable is added to quickly deploy on integration and Production. with the TTL values 14 days on dev, build and staging we need to review this again and decide on the values